packages/camera/lib/new/src/support_android/camera.dart

```diff
@@ -0,0 +1,111 @@
+part of support_android_camera;
```

Comment:
This isn't idiomatic. Any way you can avoid using it throughout this patch?

https://dart.dev/guides/libraries/create-library-packages#organizing-a-library-package